package.json

```diff
@@ -4,8 +4,8 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest",
-    "start": "node dbs/mongo/index-mongo.js"
+    "test": "jest --coverage",
+    "start": "node dbs/mongo/index.js"
```

Comment:
Maybe use nodemon? And dbs/mongo/index.js is just a script with a bunch of routes. Is this the one you want to run with npm start?

Reply:
I haven't been using the start script, I went ahead and got rid of it.  I did install nodemon but when I try and run it is says the command was not found so I didn't bother debugging it and just enter the command manually....